src/standard/utils.html

```diff
@@ -219,14 +219,14 @@
      * property.
      *
      * @method translate3d
-     * @param {HTMLElement} node Element to apply the transform to.
      * @param {number} x X offset.
      * @param {number} y Y offset.
      * @param {number} z Z offset.
+     * @param {HTMLElement} node Optional Element to apply the transform to.
```

Comment:
We should go over the whole codebase. There are plenty of "Optional in description only" type expressions. I'll fix them for this PR, just noting for future reference.